lib/tizen_emulator.dart

```diff
@@ -296,65 +292,44 @@ class TizenEmulator extends Emulator {
     @required ProcessManager processManager,
     @required TizenSdk tizenSdk,
   })  : _logger = logger,
-        _properties = properties,
-        _processUtils = ProcessUtils(
-          logger: logger,
-          processManager: processManager,
-        ),
+        _processUtils =
+            ProcessUtils(logger: logger, processManager: processManager),
         _tizenSdk = tizenSdk,
-        super(
-          id,
-          properties != null && properties.isNotEmpty,
-        );
+        super(id, properties != null && properties.isNotEmpty);
 
   final Logger _logger;
-  final Map<String, String> _properties;
   final ProcessUtils _processUtils;
   final TizenSdk _tizenSdk;
 
-  String _prop(String name) => _properties != null ? _properties[name] : null;
+  @override
+  String get name => id;
+
+  @override
+  String get manufacturer => 'Samsung';
```

Comment:
Manufacturer is 'Tizen' on emulator, except on tv-emulator.(I think this is minor things)

Reply:
I only changed the order or the properties. @HakkyuKim Do you have any opinion? It seems the manufacturer value is not available from vm_config.xml. I think it can be just null.

Reply:
I originally understood it as the literal meaning of the word manufacturer(the company that made the products). @bbrto21 Could you share any info on where to find this?

Or we can just erase the property, the default value is an empty string.

Reply:
I think he is referring to the `vendor_name` value from the `sdb capability` result.

Reply:
Indeed the result is "Tizen" for wearable emulators and "Samsung" for TV emulators. Currently I can't find the file that specifies the `vendor_name` value. We can initialize the property value by parsing the output of `sdb capability`, but the emulator has to be launched first. As another option, we can just hard-code the values differently for wearable and TV.

Or if it's not a critical issue, how about revisiting it in the future?

Reply:
> Or if it's not a critical issue, how about revisiting it in the future?

I agree with you, I think it just minor thing